test/index.test.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Hapi   = require('hapi');
+
+describe('plugin', () => {
+
+  const server = new Hapi.Server();
+  server.connection({ port: 80 });
+
+  server.register([
+    require('../lib')
+  ], (err) => {
+    /* istanbul ignore if */
+    if (err) {
+      throw err;
+    }
+  });
+
+  it('loads successfully', () => {
+    server.route([{
+      method: 'POST',
+      path: '/test',
+      config: {
+        handler: (request, reply) => {
+          reply(request.payload);
+        }
+      }
+    }]);
```

Comment:
You can setup this route outside the test. Not a big deal because this is just a dummy test for now, but thought I'd mention it.